;; Object exceptions/
;; SEMANTICDB Tags save file
(semanticdb-project-database-file "exceptions/"
  :tables
  (list
    (semanticdb-table "main.py"
      :major-mode 'python-mode
      :tags 
        '( ("myexceptions" include nil nil [1 38])
            ("base" include nil nil [39 64]))          
      :file "main.py"
      :pointmax 185
      :fsize 193
      :lastmodtime '(22017 9424 0 0)
      :unmatched-syntax '((INDENT_BLOCK 160 . 185) (COLON 158 . 159) (AS 154 . 156) (EXCEPT 137 . 143) (INDENT_BLOCK 112 . 137) (COLON 110 . 111) (TRY 66 . 69) (AS 106 . 108) ($EOI 185 . 185) (INDENT_BLOCK 171 . 196) (COLON 169 . 170) (AS 165 . 167) (EXCEPT 148 . 154) (INDENT_BLOCK 123 . 148) (COLON 121 . 122) (TRY 77 . 80) (AS 117 . 119) ($EOI 196 . 196) (INDENT_BLOCK 160 . 185) (COLON 158 . 159) (AS 154 . 156) (EXCEPT 137 . 143) (INDENT_BLOCK 112 . 137) (COLON 110 . 111) (TRY 66 . 69) (AS 106 . 108) ($EOI 185 . 185)))

    (semanticdb-table "base.py"
      :major-mode 'python-mode
      :tags 
        '( ("myexceptions" include nil nil [1 38])
            ("function" function nil nil [40 79]))          
      :file "base.py"
      :pointmax 79
      :fsize 82
      :lastmodtime '(22017 9371 0 0)
      :unmatched-syntax '(($EOI 79 . 79)))
    (semanticdb-table "myexceptions.py"
      :major-mode 'python-mode
      :tags 
        '( ("TheException" type
               (:superclasses ("Exception")
                :members 
                  ( ("pass" code nil (reparse-symbol indented_block_body) [36 40]))                  
                :type "class")
                nil [1 41]))          
      :file "myexceptions.py"
      :pointmax 41
      :fsize 42
      :lastmodtime '(22017 9179 0 0)
      :unmatched-syntax nil))
  :file "!drive_c!Users!sebastian.haas!workspace!rnd!exceptions!semantic.cache"
  :semantic-tag-version "2.0"
  :semanticdb-version "2.2")
